subprojects/gst-plugins-bad/sys/msdk/gstmsdkdec.c

```diff
@@ -1727,16 +1745,13 @@ gst_msdkdec_create_buffer_pool (GstMsdkDec * thiz, GstVideoInfo * info,
   gst_msdk_set_video_alignment (&vinfo, 0, 0, &align);
   gst_video_info_align (&vinfo, &align);
 
-  if (thiz->do_copy)
+  if (thiz->do_copy || !thiz->use_video_memory)
```

Comment:
I don't think there is a need to add " !thiz->use_video_memory' here, because thiz->do_copy is used to create other_pool used for video2system memory copy, which does not conflict with thiz->use_video_memory.

Reply:
Deleted in the last commit of this patch set, thanks.